### DISCUSSION TITLE
[Question] How do I disable the autocompletion for comments?

### DISCUSSION THREAD

**Discussion by Matixannder** [neoclide/coc.nvim#4793] [Q&A]
I'm pretty new to Neovim, but I've been liking this plugin a lot. My only question is how can I disable the autosuggestions when I'm writing inside a comment, If it is something about the specific LSP or if it's something that has to be set in the plugin config. Thanks

**Comment by fannheyward** [comment]
https://github.com/neoclide/coc.nvim/issues/407

This depends on the source, for example ` "coc.source.buffer.disableSyntaxes": ["comment"],` will disable `buffer` completions for `comment`.
